docs/src/components/layouts/utility-page-layout/utility-page-module-display.jsx

```diff
@@ -68,10 +69,10 @@ const UtilityPageModuleDisplay = ({
 				</>
 			)}
 
-			<h3>Token values and utilities</h3>
 			{/* Don't exactly know what to use this section for. */}
 			{tokens_and_utilities && (
 				<>
+					<h3>Token values and utilities</h3>
```

Comment:
This "Token values" here is not really meant to list the tokens. It is more like defining the parameters that make up a utility class/mixin and not one of the system tokens. Examples:
* `u-text(decoration)` - What is decoration? Well, it is `underline` for underlining, `strike` for strikethrough, `no-strike` for no strikethrough and `no-underline` for no underlining.
* `u-flex(direction)` - What directions can there be? Well they are `column` or `row`.

So the general rule here is to document parameters where USWDS does not link the parameter in the mixins table.

I have issues with Token here, since it is confusing based on the fact that there are real system tokens that we document separately.

This comment is not requesting a change, but should be understood as to the intent.